docs/custom-install.md

```diff
@@ -71,5 +70,72 @@ To check available configuration options, see [`deploy/cr.yaml`  :octicons-link-
         $ helm install my-db percona/psmdb-db --namespace psmdb -f values.yaml
         ```
 
+## Configure ports for MongoDB cluster components
+
+By default, the Operator starts Percona Server for MongoDB with the default port `27017` for all cluster components: `mongod`, `mongos` and `configsvrReplSet` Pods. Starting with version 1.20.0, you can start a new cluster with custom ports for all components or for a specific one. 
+
+Here's how to do it.
+
+=== "kubectl"
+
+    1. Edit the `deploy/cr.yaml` file and specify the following configuration:    
+
+        ```yaml
+        spec: 
+          ...
+          replsets:
+            - name: rs0
+              configuration: |
+                net:
+                  port: 27018
+          configsvrReplSet:
+            configuration: |
+              net:
+                port: 27019
```

Comment:
`configsvrReplSet ` is not on the same "level" with replsets and mongos. It is nested under `sharding`

```
...
  sharding:
    configsvrReplSet:
....
```

Ref: https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/cr.yaml#L361-L365

Maybe we want to specify that so that we can avoid accidental misuse of the proposed configuration.

Reply:
Updated